vocs.config.ts

```diff
@@ -27,6 +27,10 @@ export default defineConfig({
           text: "Creating a Writing Moment",
           link: "/moment/create/writing",
         },
+        {
+          text: "SalesConfig Reference",
```

Comment:
`less is more, simple is best, small is beautiful`

remove "Reference"

```suggestion
          text: "SalesConfig",
```